src/js/containers/Institutions.jsx

```diff
@@ -37,9 +37,4 @@ function mapStateToProps(state) {
   }
 }
 
-InstitutionContainer.propTypes = {
```

Comment:
For a connected component it's kinda redundant, it would be checking react-redux and react-router rather than anything we've written